sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2509,22 +2520,33 @@ object Radar {
     }
 
     /**
-     * Sends a custom event.
+     * Logs a conversion.
      *
      * @see [](https://radar.com/documentation/api#send-a-custom-event)
      *
-     * @param[customType] The name of the event.
-     * @param[metadata] The metadata associated with the event.
+     * @param[name] The name of the conversion.
+     * @param[metadata] The metadata associated with the conversion.
      * @param[callback] A callback.
      */
     @JvmStatic
-    fun sendEvent(customType: String, metadata: JSONObject?, callback: RadarSendEventCallback) {
+    fun logConversion(name: String, metadata: JSONObject? = null, callback: RadarLogConversionCallback) {
         if (!initialized) {
             callback.onComplete(RadarStatus.ERROR_PUBLISHABLE_KEY)
 
             return
         }
 
+        // if trackOnce() has been returned in the last 60 seconds, don't call it again
+        val timestampSeconds = System.currentTimeMillis() / 1000
+        val lastTrackedTime = RadarSettings.getLastTrackedTime(context)
+        val isLastTrackRecent = lastTrackedTime - timestampSeconds < 60
+
+        if (isLastTrackRecent || RadarSettings.getPermissionsDenied(context)) {
```

Comment:
This is how we do permissions checks: https://github.com/radarlabs/radar-sdk-android/blob/master/sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt#L73